### PR TITLE
Fixes #995. Nicer error styles for image errors (type + size)

### LINKED ISSUE #995: Make a nicer looking error message for "image is too big" error
Right now it looks hilariously bad, but it's still an improvement over a nginx 413 page.

@magsout, would it be possible to make a red banner with white text similar to the "Remove Image Upload" banner we have? 

(It wouldn't need to be a button, just plain text should be OK i think?)

### PR DESCRIPTION
**Pull request by miketaylr**
(We should land #1016 first, then I'll see if I need to rebase or fix any conflicts)

r? @magsout

### CODE REVIEW

File: webcompat/static/css/development/components/form.css [hunk 1]
```diff
@@ -49,10 +50,29 @@
...
+      padding: 1em 0;
+      font-weight:700;
+      background-color: var(--Form-helpMessageColor);
+      border-bottom-left-radius: 20px;
+      border-bottom-right-radius: 20px;
+      color: #fff;
+      cursor: pointer;
+      text-align: center;
+      text-decoration: underline;
+      transform: translateY(0);
```
Comment (miketaylr) [inline webcompat/static/css/development/components/form.css]: Hm, this is possibly copy-pasta that should be removed. I'm not using a `is-hidden` class to transition from.

**Comment by miketaylr** [comment]
(Redirecting review to Adam, I think @magsout is swamped at work right now -- no issues ❤️ )

r? @adamopenweb

**Comment by adamopenweb** [comment]
:+1:

**Comment by miketaylr** [comment]
Thanks @adamopenweb!

**Pull request merged**